tensorflow/core/kernels/data/interleave_dataset_op.cc

```diff
@@ -288,6 +321,25 @@ class InterleaveDatasetOp::Dataset : public DatasetBase {
       return Status::OK();
     }
 
+    Status MoveToNextElement(IteratorContext* ctx) {
```

Comment:
Annotate `EXCLUSIVE_LOCKS_REQUIRED(mu_)` to fix the build failure

Reply:
Thank you for the catch! Fixed.